edx-platform/pearson-pols-theme/lms/templates/header/header.html

```diff
@@ -40,6 +40,11 @@
 course_id = course.id if course else None
 site_status_msg = get_site_status_msg(course_id)
 %>
+
+% if configuration_helpers.get_value('GOOGLE_TAG_MANAGER_SNIPPET'):
```

Comment:
What is the default value? None? @andrey-canon

Reply:
yes